lib/openc_bot/helpers/register_methods.rb

```diff
@@ -160,7 +160,11 @@ def schema_name
       def stale_entry_uids(stale_count=nil)
         stale_count ||= default_stale_count
         sql_query = "ocdata.#{ primary_key_name } from ocdata WHERE retrieved_at IS NULL OR strftime('%s', retrieved_at) < strftime('%s',  '#{Date.today - days_till_stale}') LIMIT #{stale_count.to_i}"
```

Comment:
Can you also add "...  order by retrieved_at is null asc, retrieved_at asc" so that the stalest records are done first?